src/devices/CharacterLcd/samples/Pcf8574tSample.cs

```diff
@@ -1,7 +1,11 @@
 ﻿using System;
+using System.Device.I2c;

```

Comment:
add a license header since you're touching it (and presumably anywhere else in this dir which it might be missing)

Reply:
good point, will fix.